build.sh

```diff
@@ -144,26 +108,30 @@ fi
 if builder_start_action start:app; then
   # Start the Docker site container
 
-  if [ ! -z $(_get_docker_image_id :app) ]; then
+  if [ -d vendor ]; then
+    builder_die "vendor folder is in the way. Please delete it"
+  fi
+
+  if [ ! -z $(_get_docker_image_id app) ]; then
     if [[ $OSTYPE =~ msys|cygwin ]]; then
       # Windows needs leading slashes for path
       SITE_HTML="//$(pwd):/var/www/html/"
     else
       SITE_HTML="$(pwd):/var/www/html/"
     fi
 
-    db_ip=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' api-keyman-com-database)
+    db_ip=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ${DOCKER_IMAGE[db]})
```

Comment:
Maybe this could be a function `_get_docker_image_ip()`?

Reply:
yeah, can do as a future pr